tests/Nelmio/Alice/support/models/PropertiesParent.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Nelmio\Alice\support\models;
+
+class PropertiesParent
```

Comment:
Let's leave this as `PluralProperties` - I don't really see a benefit to making this change, and it makes it look important if it's part of the history. Also, it's marking the other class as a modification of the existing class, instead of as a new class. Just generally confusing. Make sense?